rotkehlchen/chain/ethereum/modules/convex/constants.py

```diff
@@ -0,0 +1,163 @@
+    string_to_ethereum_address('0xE689DB5D753abc411ACB8a3fEf226C08ACDAE13f'),
+    string_to_ethereum_address('0x91ad51F0897552ce77f76B44e9a86B4Ad2B28c25'),
+    string_to_ethereum_address('0x21034ccc4f8D07d0cF8998Fdd4c45e426540dEc1'),
+    string_to_ethereum_address('0x9D9EBCc8E7B4eF061C0F7Bab532d1710b874f789'),
+    string_to_ethereum_address('0xE3A64E08EEbf38b19a3d9fec51d8cD5A8898Dd5e'),
+    string_to_ethereum_address('0x8a3F52c2Eb02De2d8356a8286c96909352c62B10'),
+    string_to_ethereum_address('0x00A4f5d12E3FAA909c53CDcC90968F735633e988'),
+    string_to_ethereum_address('0x040A6Ae6314e190974ee4839f3c2FBf849EF54Eb'),
+    string_to_ethereum_address('0xbA5eF047ce02cc0096DB3Bc8ED84aAD14291f8a0'),
+    string_to_ethereum_address('0x1c86460640457466E2eC86916B4a91ED86CE0D1E'),
+    string_to_ethereum_address('0x93649cd43635bC5F7Ad8fA2fa27CB9aE765Ec58A'),
+    string_to_ethereum_address('0x5F91615268bE6b4aDD646b2560785B8F17dccBb4'),
+    string_to_ethereum_address('0xCEc9a6efFf1daF52aF12beeBf87F81bda7b95c0b'),
+    string_to_ethereum_address('0x27801399D60594BFeDe955D54c3e85B2f00179c5'),
+    string_to_ethereum_address('0xb9E2e39c9C804a01f1FCB4e86F765774D511D535'),
+    string_to_ethereum_address('0xAAf75a94394F6D06E01CcE62e2545CeFFBFA1E2D'),
+    string_to_ethereum_address('0x880c2c5c4eA8cef892a90E3f714eB60144C08c30'),
+    string_to_ethereum_address('0x08EDE581D9B9ae55FA7deCc4E4331D191BbBF9dB'),
+    string_to_ethereum_address('0x8A05801c1512F6018e450b0F69e9Ca7b985fCea3'),
+    string_to_ethereum_address('0x28a68d9c58086dAeB32d5c9297366cc91e50215D'),
+    string_to_ethereum_address('0x74835A39Fd0e72E142d5E83D514e3eF6E7642220'),
+    string_to_ethereum_address('0xE1eCBB4181378E2346EAC90Eb5606c01Aa08f052'),
+    string_to_ethereum_address('0xb83EaADa3757432f7a894944C3ac154FbdBD8B46'),
+    string_to_ethereum_address('0x834B9147Fd23bF131644aBC6e557Daf99C5cDa15'),
+    string_to_ethereum_address('0xE2585F27bf5aaB7756f626D6444eD5Fc9154e606'),
+    string_to_ethereum_address('0x28120D9D49dBAeb5E34D6B809b842684C482EF27'),
+    string_to_ethereum_address('0x92dFd397b6d0B878126F5a5f6F446ae9Fc8A8356'),
+    string_to_ethereum_address('0x19Ba12D57aD7B126dE898706AA6dBF7d6DC85FF8'),
+    string_to_ethereum_address('0x640eBdA6C4f8D70A484C170106478ffb289a47DA'),
+    string_to_ethereum_address('0xd48ad508E4e6d18bBBcA5042199e6b627dfCb331'),
+    string_to_ethereum_address('0x498Dd51c2059D7e3008067841d1efE12D6c7F99c'),
+    string_to_ethereum_address('0xe826994Cd8d4271A4b410B1b47e7DE5774Ba6E7A'),
+    string_to_ethereum_address('0x3Ac6Ef1e811bACfa283ECC25ACAD1Cb858957bAb'),
+    string_to_ethereum_address('0x5ab9Ea799904185939209c30Aa180f84AC58A63b'),
+    string_to_ethereum_address('0x8E8E06F0555CA7A33C2c1a4e01875CEbF05Bda00'),
+    string_to_ethereum_address('0x1BeCd991e1c5D0162718A215016d83E077a04cE9'),
+    string_to_ethereum_address('0xCB200C57A6ef34a2e6bBfb5E02c3e44097F259Ee'),
+]
```

Comment:
Any of those are tokens that can be owned by the user? If so we can add them to the asset list. You can create an issue here https://github.com/rotki/assets